users/tests/test_services.py

```diff
@@ -27,16 +27,20 @@ def guardian_with_children_and_enrolments():
     event4 = EventFactory(name="Event 4 for child2")
 
     occurrence1 = OccurrenceFactory(
-        event=event1, time=timezone.datetime(year=2024, month=1, day=1)
+        event=event1,
+        time=timezone.make_aware(timezone.datetime(year=2024, month=1, day=1)),
```

Comment:
How about using django.utils.timezone? https://docs.djangoproject.com/en/5.1/topics/i18n/timezones/#naive-and-aware-datetime-objects

Reply:
Okay, it seems like these make_aware calls really are needed. The timezone is aware only when django utils timezone now is called, not when the datetime is created with it.